playground/backend/internal/environment/beam.go

```diff
@@ -42,17 +42,23 @@ func NewExecutorConfig(compileCmd, runCmd, testCmd string, compileArgs, runArgs,
 
 // BeamEnvs contains all environments related of ApacheBeam. These will use to run pipelines
 type BeamEnvs struct {
-	ApacheBeamSdk  pb.Sdk
-	ExecutorConfig *ExecutorConfig
-	preparedModDir string
+	ApacheBeamSdk     pb.Sdk
+	ExecutorConfig    *ExecutorConfig
+	preparedModDir    string
+	numOfParallelJobs int
 }
 
 // NewBeamEnvs is a BeamEnvs constructor
-func NewBeamEnvs(apacheBeamSdk pb.Sdk, executorConfig *ExecutorConfig, preparedModDir string) *BeamEnvs {
-	return &BeamEnvs{ApacheBeamSdk: apacheBeamSdk, ExecutorConfig: executorConfig, preparedModDir: preparedModDir}
+func NewBeamEnvs(apacheBeamSdk pb.Sdk, executorConfig *ExecutorConfig, preparedModDir string, numOfParallelJobs int) *BeamEnvs {
+	return &BeamEnvs{ApacheBeamSdk: apacheBeamSdk, ExecutorConfig: executorConfig, preparedModDir: preparedModDir, numOfParallelJobs: numOfParallelJobs}
 }
 
 // PreparedModDir returns the path to the directory where prepared go.mod and go.sum are located
 func (b *BeamEnvs) PreparedModDir() string {
 	return b.preparedModDir
 }
+
+// NumOfParallelJobs returns the max number of the possible code processing for the SDK on that instance at the same time
```

Comment:
// NumOfParallelJobs returns the max number of the possible code executions on the instance simultaneously.

Reply:
Done.